web/src/engine/osk/src/keyboard-layout/gesturePreviewHost.ts

```diff
@@ -4,6 +4,8 @@ import EventEmitter from "eventemitter3";
 import { KeyElement } from "../keyElement.js";
 import { FlickNameCoordMap, OrderedFlickDirections } from "../input/gestures/browser/flick.js";
 import { PhoneKeyTipOrientation } from "../input/gestures/browser/keytip.js";
+import { default as VisualKeyboard } from "../visualKeyboard.js";
+import { renameSpecialKey } from "./oskKey.js";
```

Comment:
`default` seems to be unused in this file.

```suggestion
import { renameSpecialKey } from "./oskKey.js";
```

Reply:
No, it _very_ much is - just via the alias name `VisualKeyboard`.  Note:

https://github.com/keymanapp/keyman/blob/779196e1fd9dca90f9c7834b400a222509fd2ffe/web/src/engine/osk/src/visualKeyboard.ts#L124

`export default class FooBar` results in a class known as `FooBar` within its file, but known as an export _outside_ the class as `default`.  As in, `default` is the literal name given to it for imports elsewhere.

`import VisualKeyboard from 'file.js'` is the same as `import { default as VisualKeyboard } from 'file.js'`.  It's just a bit more exposed.